src/frontend/Layout/Header/header.css

```diff
@@ -10,6 +10,9 @@ text-decoration: none;
 .header-bar {
   background-color: var(--accent--one-color);
   color: var(--foreground-color);
+  z-index:999;
```

Comment:
be specific about giving z index. May cause issues later.

Reply:
Rightly pointed out by @vijaytembugade!